superset-frontend/src/SqlLab/components/SqlEditorLeftBar/index.tsx

```diff
@@ -113,7 +113,7 @@ const SqlEditorLeftBar = ({
     'schema',
   ]);
 
-  const [emptyResultsWithSearch, setEmptyResultsWithSearch] = useState(false);
+  const [_emptyResultsWithSearch, setEmptyResultsWithSearch] = useState(false);
```

Comment:
Do we actually need this?

Reply:
I think we need `setEmptyResultsWithSearch`, and are flushing position 1 of the array. Code is a bit cryptic to me here. Not sure why we need `setEmptyResultsWithSearch`

Reply:
Since it's meant to be flushed, I could put it in a `_` to make it more clear (and keep the spread operator) or hard reference `useState(false)[1]`, WDYT?

Reply:
I am not sure that we need this. It looks that originally the [PR that introduced this ](https://github.com/apache/superset/pull/19598/files#diff-bbf91464669d98b9e120f57ce26155804f7238a277fcc30c0d805a943f6c73edR190) was just meant to show some alternate text. I did some testing and I could not see anything breaking with just removing this entirely.